src/code_80057C60.c

```diff
@@ -6275,16 +6179,13 @@ void render_balloon(Vec3f arg0, f32 arg1, s16 arg2, s16 arg3) {
     gDPLoadTextureBlock(gDisplayListHead++, D_8018D4BC, G_IM_FMT_CI, G_IM_SIZ_8b, 64, 32, 0, G_TX_NOMIRROR | G_TX_CLAMP, G_TX_NOMIRROR | G_TX_CLAMP, G_TX_NOMASK, G_TX_NOMASK, G_TX_NOLOD, G_TX_NOLOD);
     gSPVertex(gDisplayListHead++, D_800E5250, 4, 0);
     gSPDisplayList(gDisplayListHead++, D_0D008C78);
-    // Not sure what to make of the `- 0x40`, it matches, but it looks weird to me
+    // TODO: Is this subraction `- 0x40` fake? it matches, but it's not right, should be D_8018D480
     gDPLoadTextureBlock(gDisplayListHead++, D_8018D4C0 - 0x40, G_IM_FMT_CI, G_IM_SIZ_8b, 64, 32, 0, G_TX_NOMIRROR | G_TX_CLAMP, G_TX_NOMIRROR | G_TX_CLAMP, G_TX_NOMASK, G_TX_NOMASK, G_TX_NOLOD, G_TX_NOLOD);
```

Comment:
I wonder if 8018D4C0 is part of an array in an earlier variable?

Reply:
Made a clear comment about it, might add some AVOID_UB define in another PR.